aws-cli/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM python:3.7-alpine
```

Comment:
Pin also the alpine version

Reply:
Also I'd add yourself as maintainer so we know that you did it and probably you will be the one indicated to look into it if there are issues

Reply:
recently I started not pinning alpine, there's no major differences between minor versions and the official images updated `alpine` to point to the latest alpine version again